dune/grid/common/ZoltanGraphFunctions.cpp

```diff
@@ -167,7 +167,7 @@ void getNullEdgeList(void *cpGridPointer, int sizeGID, int sizeLID,
     (void) cpGridPointer; (void) sizeGID; (void) sizeLID; (void) numCells;
     (void) globalID; (void) localID; (void) numEdges; (void) nborGID;
     (void) nborProc; (void) wgtDim; (void) ewgts;
-    err = ZOLTAN_OK;
```

Comment:
One possible mechanism is if `ZOLTAN_OK` is `#define`d as zero (can't check right now) in some header.  In that case the statement
```C++
err = ZOLTAN_OK;
```
makes `err` into a null pointer within the function body and there's no change visible in the caller.  If, moreover, the caller already initialises `*err` as `ZOLTAN_OK` before calling into the call-back then the call-back is a big no-op.